src/Components/Heic.tsx

```diff
@@ -33,6 +33,12 @@ export const Heic = forwardRef<HTMLImageElement, HeicProps>(
             (async () => {
                 try {
                     const res = await fetch(url);
+
+                    if (res.headers.get('content-type') === 'image/jpeg') {
```

Comment:
We should probably have a comment here as to why this is the way it is

Specifically, this is due to an assumption based on how the other components in the system behave

An heic is either an heic or a compressed jpeg